### DISCUSSION TITLE
Error in Copilot Modal Box: Unable to Cancel Progress - Need Assistance

### DISCUSSION THREAD

**Discussion by 0xTanzim** [community/community#72905] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello,

I am facing an issue with running copilote. When I try to cancel it, a box is displayed and I am unable to cancel my progress. I would like to hear your suggestions on how to resolve this issue. 

Also, I have attached a screenshot for you to look over. 

Thank you.

<hr>

![Screenshot 2023-10-30 061305](https://github.com/community/community/assets/60542516/71a25ac5-5cce-4c96-9fca-a22fd049876e)

**Comment by b8zeek** [comment, marked as answer]
Greetings, check this comment, follow the steps and it should solve your issue.

https://github.com/orgs/community/discussions/73087#discussioncomment-7440076

Cheers.

🍀
